frappe/tests/test_website.py

```diff
@@ -4,13 +4,19 @@
 
 import frappe
 from frappe.website import render
+from frappe.website import serve
```

Comment:
```suggestion
from frappe.website.serve import get_response
```

The `serve.get_response()` below just sounds wrong. Like `serve` would contain a badly named _Server_ class. 😄